smart_contract_examples/SwapExamples.sol

```diff
@@ -10,12 +10,12 @@ contract SwapExamples {
     // we will detail the design considerations when using
     // `exactInput`, `exactInputSingle`, `exactOutput`, and  `exactOutputSingle`.
 
-    // It should be noted that for the sake of these examples, we purposefully pass in the swap router instead of inherit the swap router for simplicity.
+    // It should be noted that for the sake of these examples, we purposefully pass in the swap router instead of inheriting the swap router for simplicity.
```

Comment:
Original: 

> It should be noted that for the sake of these examples, we purposefully pass in the swap router instead of inheriting the swap router for simplicity.

Shouldn't this be ...?:

> It should be noted that for the sake of simplicity, we purposefully pass in the swap router instead of inheriting the swap router in these examples.

Reply:
this looks good